page-objects/lib/toast.po.ts

```diff
@@ -0,0 +1,19 @@
+import type {Locator} from '@playwright/test';
+import {BasePO} from '@agnos-ui/base-po';
+
+export const toastSelectors = {
+	rootComponent: '.toast',
+	closeButton: '.btn-close',
+};
+
+export class ToastPO extends BasePO {
+	selectors = structuredClone(toastSelectors);
+
+	override getComponentSelector(): string {
+		return this.selectors.rootComponent;
+	}
+
+	get locatorCloseButton(): Locator {
+		return this.locatorRoot.locator(this.selectors.closeButton);
+	}
```

Comment:
I think there is a missing state() method, to return the displayed state of the widget. All revelant information, like in the rating.po for example.

For example, the state could return:

- The class applied,
- The close button (undefined if no button)
- The aria-label for the close button (undefined if no button)
- The header text (undefined if no header)
- The body text

The issue is that the toast (this is the same for the alert, at the time this architecture was not in place) is not well tested form an e2e point of view (we don't test that everything is "connected").
The current e2e test is fine, but it tests only the containers, not the toasts themselves.

To test the toast with different configurations, no need to add samples, you can use the playground. Even if it doesn't work fully, it allows to play with the props easily. 

For example, you can check the returned state for 

`/components/toast/playground#{"props":{"slotDefault":"This is a toast","autohide":false,"className":"text-bg-primary"}}`, 

then change the hash for 

`components/toast/playground#{"props":{"slotDefault":"This is a toast","autohide":false,"className":"text-bg-primary","slotHeader":"header"}}`

And check the changes compared to the expected state. No need to check deeply (this is the UT purposes), but at least go through all the concept once to check that everything is "plugged".